### ISSUE TITLE
Unable to test which cursor is used

### ISSUE THREAD

**Issue by gsnedders** [w3c/web-platform-tests#7750]
At the moment, we have no way to test the CSS `cursor` property. We probably need some ability to take a screenshot of the cursor at a specific place.

cc/ @frivoal

**Comment by frivoal** [comment]
Right. Currently all cursor tests are manual, and that's lousy.

**Comment by RByers** [comment]
FWIW, years ago I added a mechanism in chromium to read back the cursor state (including custom bitmaps).  I think that's probably more practical than doing screenshot diffing - at least for the OS defined cursors.

**Comment by gsnedders** [comment]
@RByers There are tests for things like colour space and size of image cursors, which means you need more than just state.

**Comment by foolip** [comment]
It seems like in order to automate this, you'd need to take a screenshot of the whole desktop environment. A handful of manual tests for this might be an OK tradeoff :)

**Comment by frivoal** [comment]
Indeed. Image cursors would be quite a pain to test, but even worse are the various predefined cursors, that don't even have a set shape, and need human judgement to decide if the cursor is indeed a cursor that "Indicates something is to be copied. Often rendered as an arrow with a small plus sign next to it."

If someone has a bright idea, that'd be great, but otherwise, we can certainly focus our energy better somewhere else.
